saleor/graphql/product/mutations/products.py

```diff
@@ -802,6 +802,18 @@ def clean_attributes(
     @classmethod
     def clean_input(cls, info, instance, data):
         cleaned_input = super().clean_input(info, instance, data)
+
+        weight = cleaned_input.get("weight")
+        if weight and weight.value < 0:
+            raise ValidationError(
+                {
+                    "weight": ValidationError(
+                        "Product can't have negative weight.",
+                        code=ProductErrorCode.INVALID,
```

Comment:
should be `ProductErrorCode.INVALID.value`?

Reply:
I try to use ErrorCode without `.value` in all places where `mypy` don't rise error.

Reply:
We shouldn't use `.value` if `mypy` don't raise an error.